test/residency/prototypes/test_task015_hatchery_attribution_public_seam.py

```
@@ -0,0 +1,927 @@
+        f"native.host_memory={'observed' if host_memory else 'unknown'}",
+        f"native.boot_identity={'observed' if boot_identity else 'unknown'}",
+        f"native.device_identity={'observed' if observed else 'unknown'}",
+        "native.process_fdinfo=unknown",
+        "native.cgroup_membership=unknown",
+        f"hatchery.native_causal_attribution={'fallback' if observed else 'deferred'}",
+    ]
+
+
+def require_native_profile_classifier() -> None:
+    require(
+        classify_hatchery_identity("hatchery", True) == "exact",
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)